op-chain-ops/genesis/genesis.go

```diff
@@ -95,6 +96,17 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 	}, nil
 }
 
+func regolithTime(config *DeployConfig, block *types.Block) *uint64 {
```

Comment:
What do you think about making this a function on the `DeployConfig` itself?

```go
func (cfg *DeployConfig) RegolithTime(block *types.Block) *uint64 {}
```

Might make it more portable + we could easily add a unit test this way in the file that parses deploy config json

Reply:
Yeah makes sense to me. I'm not 100% sure if we'll ultimately want to specify RegolithTime as an offset from genesis but it's certainly very helpful in tests currently so seems like a good starting point until we know otherwise.